core/src/subgraph/runner.rs

```diff
@@ -723,7 +725,7 @@ where
         for trigger in triggers {
             // Using an `EmptyStore` and clearing the cache for each trigger is a makeshift way to
             // get causality region isolation.
-            let schema = self.inputs.store.input_schema();
+            let schema = ReadStore::input_schema(&self.inputs.store);
```

Comment:
Why was that change necessary?

Reply:
I added the `input_schema` function to another trait which caused this call to be ambiguous.